Mathlib/Data/Finset/Basic.lean

```diff
@@ -1803,21 +1803,45 @@ theorem inter_left_idem (s t : Finset α) : s ∩ (s ∩ t) = s ∩ t := inf_lef
 theorem inter_right_idem (s t : Finset α) : s ∩ t ∩ t = s ∩ t := inf_right_idem _ _
 #align finset.inter_right_idem Finset.inter_right_idem
 
+theorem inter_union_distrib_left (s t u : Finset α) : s ∩ (t ∪ u) = s ∩ t ∪ s ∩ u :=
+  inf_sup_left _ _ _
+#align finset.inter_union_distrib_left Finset.inter_union_distrib_left
```

Comment:
Don't create `#align`s! They correspond to Lean 3 declarations and `finset.inter_union_distrib_left` never was a Lean 3 declaration.
```suggestion
#align finset.inter_distrib_left Finset.inter_union_distrib_left
```
and below